forwards_paper.tex

```diff
@@ -186,8 +187,7 @@ \subsection*{Tree sequences}
         edges that define these topologies. The right-hand panels show the specific encoding
         of this tree sequence in the four tables (nodes, edges, sites and mutations) defined
         by \msprime.
-        \jk{The figure needs to be rescaled a bit to give more space to the trees. We could also use a
-        time scale next to the trees.}
+        \jk{Some minor alignment issues present here.}
```

Comment:
looks pretty good to me.  I can always do final tweaks in inkscape if we want.

Reply:
I'll twiddle the alignments of a few columns at some point, but I think it's basically OK for now.